pkg/autoscaler/config.go

```diff
@@ -29,10 +29,6 @@ import (
 const (
 	// ConfigName is the name of the config map of the autoscaler.
 	ConfigName = "config-autoscaler"
-
-	// Default values for several key autoscaler settings.
-	DefaultStableWindow           = 60 * time.Second
```

Comment:
I'm not sure removing these constants is  an improvement, if we just want to not expose them why not make them private? In general I think its much nicer to have a constant like this so folks dont have to dig through source when looking for the value.

Reply:
I had the same thought and an initial version of this pulled all the constants up in the block. However, all other configs seem to not do that so I opted to remove them here for consistency as well.